test/test_ops.py

```diff
@@ -672,7 +768,7 @@ def test_iou(self):
         # Check if all elements of tensor are as expected.
         assert out.size() == torch.Size([3, 3])
         tolerance = 1e-4
-        assert ((out - expected).abs().max() < tolerance).item() is True
+        assert ((out - expected).abs().max() < tolerance).item()
```

Comment:
Reverting this change as suggested by @pmeier in https://github.com/pytorch/vision/pull/2710#discussion_r495760308 . Let's do this in a separate PR
```suggestion
        assert ((out - expected).abs().max() < tolerance).item() is True
```

Reply:
Reverted